tests/WebexMeeting.e2e.js

```diff
@@ -15,7 +15,6 @@ describe('Meeting Widget', () => {
         user = browser.call(() => createUser());
         accessToken = user.token.access_token;
       }
-
```

Comment:
any reason for removing the new lines here and [here](https://github.com/webex/widgets/pull/96/files#diff-f5d4a1540edc79b8cc150656aaa0a676d13340692408b2aa279a92c96ec973e9L43)?

Reply:
I removed the spaces on line 28
I thought there was no need that empty space